backend/tests/use_cases/test_user_interactor.py

```diff
@@ -1,11 +1,9 @@
 import pytest
 from unittest.mock import MagicMock
 from app.repositories.user_repository import UserRepository
-from app.use_cases.user_interactor import (
-    register_user_interactor,
-    login_user_interactor,
-    change_password_interactor
-)
+from app.use_cases.register_user_interactor import RegisterUserInteractor
+from app.use_cases.login_user_interactor import LoginUserInteractor
+from app.use_cases.change_password_interactor import ChangePasswordInteractor
```

Comment:
Is the unit testing in this file covering most of the user interactor?

Reply:
Yes, the unit testing in this file covers most of the key scenarios for each user interactor, including handling errors and validating input for registration, login, and password change.